### ISSUE TITLE
trips API text color

### ISSUE THREAD

**Issue by GerdC** [motis-project/motis#664]
I'm playing around with the Motis 2 trips openapi.

Sometimes, it returns a routeColor, which is nice to give markers a color as demonstrated on https://europe.motis-project.de.

**I'd like to propose to also return a textColor property**, which should be available whenever a routeColor is available.

In the default Motis2 UI, the missing text color isn't obvious, as the rendered vehicle markers contain no textual clue about the route or mode.

As a user, I don't like to play memory on maps.

**Comment by felixguendling** [comment]
I agree, this can be added. Feel free to open a PR if you need it urgently.

**Comment by vkrause** [comment]
Isn't this what `routeTextColor` provides already (if set in GTFS)?

**Comment by GerdC** [comment]
Code could be here
https://github.com/motis-project/motis/blob/42de676ea151c38c94f49ab762fdf9a9b2f92b41/src/railviz.cc#L377

probably something like
to_str(from.get_route_color(nigiri::event_type::kDep).text_color_)

**Comment by felixguendling** [comment]
Yes, this would be the code change. Additionally the openapi.yml spec needs to be adjusted.
